docs/install.md

```diff
@@ -802,15 +804,15 @@ the new default behavior.
 The following commands trigger messages that have been changed from `stdout`
 print calls to `INFO`-level (or `WARNING`- or `ERROR`-level) log messages:
 
-```
+```screen
```

Comment:
Seems like this block and the next one would make more sense as a list of fixed-width items.